extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -165,9 +165,7 @@ export class PublishOptionsDialog {
 
 		await this.updateOptionsTable();
 		this.optionsFlexBuilder?.removeItem(this.optionsTable!);
-		this.optionsFlexBuilder?.insertItem(this.optionsTable!, 0, { CSSStyles: { 'overflow': 'scroll', 'height': '65vh' } });
-
-		TelemetryReporter.sendActionEvent(TelemetryViews.PublishOptionsDialog, TelemetryActions.resetOptions);
```

Comment:
why is this being removed?

Reply:
Again un-intentional, added it back, If you know how can i check using git blame, can you tell me please, Thanks :)

Reply:
What do you want to check with git blame? That isn't going to tell you specifics of why something was removed - just tell you in what commit it was removed/modified. And since this is all in your branch then it has to be something you did - whether intentionally or not.